data/human/news.txt

```diff
@@ -137,6 +149,11 @@ news "merchant in human space"
 			"Some Flat Earthers accused me of playing propaganda videos on the ship's viewports when they looked out of the ship while we were in orbit."
 			"There's no point crying over spilt milk. But the bill for cleaning the 200 tons of milk that leaked from my ship is no laughing matter either."
 			"I spy with my little eye... something large, flat, and sturdy. The landing pads! How did you know?"
+			"A Starbarge, by any other name, is still just a moving cargo container."
+			"I named my ship Titanic. It's a strong sounding name, but I feel people are making fun of me."
+			"The early Firebird gets the Wormhole."
```

Comment:
This doesn't really feel like a spoiler to me (humanity knows about the theory of Wormholes since Albert Einstein wrote about them in 1915), but this news-item does work better in a Hai context, so I moved it to `"hai news.txt"`.